e2e/native-async-mock/babel.config.js

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
 
 module.exports = {
-  plugins: ['@babel/plugin-transform-async-to-generator'],
-  presets: ['@babel/preset-env'],
+  presets: [['@babel/preset-env', {targets: {node: 8}}]],
```

Comment:
not sure about this one, since we test native async, do we even need babel here?

Reply:
do we test it? How can that pass on node 6?

Reply:
the test is conditionally run for Node > 7.6.0

Reply:
Ah, ok. Nah, should probably remove it, or at least blacklist the async transform